src/internal/m365/collection/exchange/contacts_container_cache.go

```diff
@@ -124,8 +129,10 @@ func (cfc *contactContainerCache) Populate(
 	}
 
 	if err := cfc.populatePaths(ctx, errs); err != nil {
-		return clues.Wrap(err, "populating paths")
+		return clues.WrapWC(ctx, err, "populating paths")
 	}
 
+	logger.Ctx(ctx).Info("done populating container cache")
```

Comment:
Suggestion, incase we're looking for runtime heuristics:
```suggestion
        // top of func
        start := time.Now()
        // here
	logger.Ctx(ctx).With("duration", time.Now().Sub(start)).Info("done populating container cache")
```